aiida_wannier90/parsers.py

```diff
@@ -105,8 +105,10 @@ def parse(self, **kwargs):
         if exiting_in_stdout:
             return self.exit_codes.ERROR_EXITING_MESSAGE_IN_STDOUT
 
+        return self.exit_codes.ExitCode(0)
```

Comment:
I guess this is the change that causes the tests to fail.

Reply:
I was getting the following error: 

_aiida_wannier90/parsers.py
  Line: 30
    pylint: inconsistent-return-statements / Either all return statements in a function should return an expression, or none of them should. (col 4)_

So, either I disable the pylint exception or I was looking a for trivial ExitCode . But maybe something change concerning this specific exit code...

Reply:
@sphuber can answer definitively, but I think in this case it's fine not to return anything.

To be honest I'm not too fond of this check, for all I care we could disable it globally as in https://github.com/aiidateam/aiida-quantumespresso/blob/develop/.pylintrc#L64

Reply:
You can return an exit code with status 0 as that has the same effect as returning `None`, but it should be simply `return ExitCode()`. The default value is `0` but you can of course make it explicit `return ExitCode(0)`. The `self.exit_codes` is a property to a mapping of the defined exit codes on the calculation class. So you can only use the exit code labels used in their definition, e.g. if the `CalcJob` spec contains `spec.exit_code(418, 'ERROR_I_AM_A_TEAPOT') then you can get it with `self.exit_codes.ERROR_I_AM_A_TEAPOT`, which will simply return `ExitCode(418, 'ERROR_I_AM_A_TEAPOT')`.

Reply:
But yes, I also typically just ignore this warning because I think in these cases I think it is fine to have either `None` or an `ExitCode` as return statement

Reply:
Ok, thank´s both of you!
Maybe for the moment I can disable it locally and for the future we could do something similar to quantum espresso? @giovannipizzi  what do you think?

Reply:
Yeah, let's just ignore it locally for now.

Any of the proposed solutions would be ok, really -- since this doesn't influence behavior or interface, it's not really important which one we choose.

Reply:
Ok to disable it, for sure for the file!